src/components/DataDisplay/Table.tsx

```diff
@@ -69,7 +69,7 @@ export default function Table({ data, globalFilter }: { data: object[]; globalFi
                 return columnHelper.accessor(header, {
                     // eslint-disable-next-line react/no-danger, react/no-unstable-nested-components
                     cell: (info) =>
-                        info.getValue() && (
+                        info.getValue() && ( // TODO: only add a hyperlink if the specification MPEG entry is true
```

Comment:
@DenizUgur hi Deniz, could you maybe help with this one? I would like to make sure that only MPEG specs are forwarded to FFC. I already got rid of the specifications.csv and replaced it with a json which has the MPEG boolean field. I think this is now pretty easy to add.

Reply:
See #190